src/components/RedigerAvansertBeregning/hooks.tsx

```diff
@@ -64,6 +60,45 @@ export const useFormLocalState = (initialValues: {
       : undefined
   )
 
+  const minAlderForHeltUttak = React.useMemo(() => {
+    if (
+      localGradertUttak?.uttaksalder?.aar &&
+      localGradertUttak?.uttaksalder?.maaneder !== undefined
+    ) {
+      const gradertAlder = { ...localGradertUttak.uttaksalder } as Alder
+      const localGradertUttakPlus1Maaned = getAlderPlus1Maaned(gradertAlder)
+      // if the previously chosen uttaksalder is lower than localGradertUttakPlus1Maaned
+      if (
+        localHeltUttak?.uttaksalder?.aar &&
+        localHeltUttak?.uttaksalder?.maaneder !== undefined &&
+        localHeltUttak.uttaksalder.aar * 12 +
+          localHeltUttak.uttaksalder.maaneder <
+          localGradertUttakPlus1Maaned.aar * 12 +
+            localGradertUttakPlus1Maaned.maaneder
+      ) {
+        setHeltUttak((previous) => ({
+          ...previous,
+          uttaksalder: undefined,
+        }))
+      }
+      return localGradertUttakPlus1Maaned
+    } else {
+      return undefined
+    }
+  }, [localGradertUttak])
+
+  const maxAlderForGradertUttak = React.useMemo(() => {
+    if (
+      localHeltUttak?.uttaksalder?.aar &&
+      localHeltUttak?.uttaksalder?.maaneder !== undefined
+    ) {
+      const heltAlder = { ...localHeltUttak.uttaksalder } as Alder
+      return getAlderMinus1Maaned(heltAlder)
+    } else {
+      return { aar: 74, maaneder: 11 }
```

Comment:
Antar dette er "default" alder for helt uttak (75 år) minus 1 måned.
Men en uinnvidd vil nok lure på hva som er spesielt med 74 år 11 måneder.

For å gjøre det litt klarere er mitt forslag:
```
const DEFAULT_HELT_ALDER = { aar: 75, maaneder: 0 }

const maxAlderForGradertUttak = React.useMemo(() => {
    const alderIsDefined = localHeltUttak?.uttaksalder?.aar &&
      localHeltUttak?.uttaksalder?.maaneder !== undefined
    const heltAlder = alderIsDefined ? { ...localHeltUttak.uttaksalder } as Alder : DEFAULT_HELT_ALDER
    return getAlderMinus1Maaned(heltAlder)
  }, [localHeltUttak])
```
(eller noe lignende :)

Reply:
Enig! Jeg har denne constant fra før av (DEFAULT_SENEST_UTTAKSALDER) som er 75 år og 0 md.
Jeg trekker heller én måned fra den som du foreslår.